diff --git a/scripts/rft_yaml_to_junit_xml b/scripts/rft_yaml_to_junit_xml
index feb6463..2d3b3a3 100755
--- a/scripts/rft_yaml_to_junit_xml
+++ b/scripts/rft_yaml_to_junit_xml
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby
 
 # Copyright (c) 2015 Cisco Systems, Inc.  All rights reserved.
 #
diff --git a/scripts/runfabtests.sh b/scripts/runfabtests.sh
index b57a039..9b3f0e7 100755
--- a/scripts/runfabtests.sh
+++ b/scripts/runfabtests.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
 #
 # Copyright (c) 2017, Intel Corporation.  All rights reserved.
